### DISCUSSION TITLE
lotus: symbol lookup error

### DISCUSSION THREAD

**Discussion by uranuschen365** [filecoin-project/lotus#10315] [Build and Sync]
I compiled lotus v1.20.0-rc2 version,but the program execution result reported an error：./lotus: symbol lookup error: ./lotus: undefined symbol: fcntl64. my go version is 1.19

**Comment by rjan90** [comment]
Hey @uranuschen365! Can you share the steps that you did before the error? Are you seeing lotus being installed in your system when checking `lotus -v`?

**Reply by uranuschen365** [reply]
I compiled in another machine and then it was successful to startup. I think my os lacks some environment.

**Comment by codex8080** [comment]
I have met the same problem, can someone can tell me some solution?
